### DISCUSSION TITLE
Very light pen strokes registering as holding middle mouse button and dragging

### DISCUSSION THREAD

**Discussion by ranguli** [OpenTabletDriver/OpenTabletDriver#3374] [Q&A]
## Hardware
- Wacom CTH-470
-  Pop!_OS 22.04 LTS
- OpenTabletDriver v0.6.4.0  (official .deb)

## Settings
I have the output mode is set to "Artist Mode", the Tip Binding is set to `Linux Artist Mode: (Button 1: Pen Button 1)`.

## Issue
While "normal" pressure brush strokes are correctly registered and give the expected result for painting in applications such as GIMP, if a very light pressure is used and the pen is moved, it registers as the middle mouse button being held and the mouse being dragged. For example, it is causes panning in GIMP or Blender. Variations in drawing pressure can accidentally invoke the panning binding while trying to draw brushstrokes.

Most importantly, I can't see anywhere in OTD where this setting would be enabled or disabled.

## Expected Behavior
Bindings that are conditional to pressure sensitivity are visible and can be toggled in OTD.

**Comment by ranguli** [comment]
I'm not sure if this constitutes a bug or is known/intended behavior. I will reopen it as an Issue if this is not supposed to be happening or can't be configured.

**Comment by jamesbt365** [comment]
OTD processes stuff in a pretty standard way, if data is being parsed correctly within the `tablet > tablet debugger` this likely lies on the applications or window manager causing this behaviour

**Reply by ranguli** [reply]
I should add that I'm using Gnome on X11
